annotation_full.wdl

```diff
@@ -6,12 +6,13 @@ import "./functional-annotation.wdl" as fa
 workflow annotation {
 input {
   String  proj
-  String  input_file
+  File  input_file
   String  imgap_project_id
   String  database_location="/refdata/img/"
   String  imgap_project_type="metagenome"
-  String?  gm_license="/refdata/licenses/.gmhmmp2_key"
```

Comment:
I would like this left as optional, if a user has a license file in their home dir and that is in their env a license file doesn't need to be copied in.

Reply:
I believe that by having a default it available as an optional default they can still set the variable. The question mark is then redundant.